packages/cli/src/build/__tests__/basic.ts

```diff
@@ -366,6 +549,66 @@ test("does not duplicate babel helpers when using @babel/plugin-transform-runtim
   `);
 });
 
+test("imports helpers for a helper only available in a newer version of @babel/runtime (without @babel/plugin-transfrom-runtime)", async () => {
+  const dir = await testdir({
+    "package.json": JSON.stringify({
+      name: "test",
+      main: "dist/test.cjs.js",
+      dependencies: {
+        "@babel/runtime": "7.9",
+      },
+    }),
+    "babel.config.json": JSON.stringify({
+      presets: [require.resolve("@babel/preset-env")],
+    }),
+    "src/index.js": "import './other'; for (const x of something) {}",
+    "src/other.js": "for (const x of something) {}",
+  });
+  await build(dir);
+  expect(await getDist(dir)).toMatchInlineSnapshot(`
+    ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ dist/test.cjs.dev.js, dist/test.cjs.prod.js ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
+    'use strict';
+
+    var _createForOfIteratorHelper = require('@babel/runtime/helpers/createForOfIteratorHelper');
```

Comment:
shouldnt u add a very similar test that would check that this helper has been inlined rather than required?

Reply:
As in when the version of @babel/runtime doesn't include that helper? That test already exists before this PR - https://github.com/preconstruct/preconstruct/blob/b9e91373985b967daa6440ba15970e3e8dd776d2/packages/cli/src/build/__tests__/basic.ts#L238-L367

Reply:
a similar test - yes, but without `@babel/plugin-transform-runtime` being used